README.md

```diff
@@ -48,13 +48,19 @@ To start contributing, first look over our [onboarding](https://github.com/Donal
 
 A list of our group:
```

Comment:
Maybe we should mention somewhere that we are adopting this project from CSE 110 and differentiate those contributors from our current group.